compiler/src/iree/compiler/Codegen/Dialect/UKernelOps.cpp

```diff
@@ -76,14 +76,12 @@ std::pair<int64_t, int64_t> UKernelGenericOp::getDpsInitsPositionRange() {
 
 /// Map type of operand of a `iree_codegen.ukernel.generic` operation to
 /// the type(s) of the function call arguments(s) it lowers to.
-static LogicalResult getCallOpType(MLIRContext *context,
-                                   Type microKernelOpOperandType,
-                                   SmallVector<Type> &callOperandTypes) {
+static LogicalResult lowerToPointerTypes(MLIRContext *context,
```

Comment:
This seems like it is changing the lowering for all pointer types..... It is being changed from `ptr, offset, sizes, strides` to just `ptr, offset`.... I thought that the consensus was that `ukernel.generic` will still lower to `ptr, offset, sizes, strides`....

Reply:
And it does, because after this `lowerToPointerTypes` function, we have `lowerToNonPointerTypes` where all the other arguments are appended. 

Maybe start reviewing this PR by the `.mlir` test files. This way you'll see first that all this PR really does is reorder arguments, it does not add or remove arguments.

The naming of these helpers is clearly not optimal (probably what caused the confusion). I went for minimum diff here. I am planning a sequence of PRs, one of which will clean up these helpers and be the place to think about names.